server/package.json

```diff
@@ -4,34 +4,40 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "start": "webpack --colors --progress",
-    "build": "",
-    "lint": "eslint \"./**/*.js\""
+    "start": "node dist/server.js",
+    "dev": "webpack --config webpack.dev.js --colors --progress",
+    "clean": "rm -rf dist/",
+    "build": "yarn clean && webpack --config webpack.prod.js",
+    "postinstall": "yarn build"
+  },
+  "engines": {
+    "node": "10.14.1",
```

Comment:
This needs to be: `"node": "10.4.2",`

Reply:
oh i thought we were using 10.14.1. cool! super easy fix 💯